src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/tts_get_capabilities_request.cc

```diff
@@ -57,6 +57,11 @@ void TTSGetCapabilitiesRequest::Run() {
   SendRequest();
 }
 
+void TTSGetCapabilitiesRequest::onTimeOut() {
+  LOG4CXX_AUTO_TRACE(logger_);
+  hmi_capabilities_.RemoveFromDefaultInitializedCapabilities(
```

Comment:
@LuxoftAKutsan in case if some capabilities initialized by default, SDL should send the appropriate requests to the HMI and need to wait for all responses, so we put all function ids of requests that SDL sent to the HMI in the **default_initialized_capabilities_** set and wait for responses. When all responses received - the _hmi_cooperating_ flag should be changed to **true**. But in case if HMI doesn't respond - capabilities still will have the default values and we still should change the _hmi_cooperating_ flag to true. This is why we need to call this function _RemoveFromDefaultInitializedCapabilities_ - just to remove the function id from the **default_initialized_capabilities_** set and when the set will be empty - we will change the _hmi_cooperating_ flag  value to **true**